lib/gnat/jetstream/api/kv.ex

```diff
@@ -248,10 +251,34 @@ defmodule Gnat.Jetstream.API.KV do
     Gnat.Jetstream.API.KV.Watcher.stop(pid)
   end
 
+  @doc ~S"""
+  Returns true if operation is `{"kv-operation", "DEL"}` or `{"kv-operation", "PURGE"}`
+
+  ## Parameters
+  - `headers` - a list of headers to test
+
+  ## Examples
+
+      iex> is_delete_operation?([{"kv-operation", "DEL"}])
+      true
+      iex> is_delete_operation?([{"kv-operation", "PURGE"}])
+      true
+      iex> is_delete_operation?([{"kv-operation", "ADD"}])
+      false
+  """
+  @spec is_delete_operation?(headers :: Gnat.headers()) :: boolean()
+  def is_delete_operation?(headers) do
```

Comment:
```suggestion
  @spec delete_operation?(headers :: Gnat.headers()) :: boolean()
  def delete_operation?(headers) do
```

As an Elixir convention, using the `?` is correct for a function that returns a boolean, but the `is_` prefix is reserved for guards. See https://hexdocs.pm/elixir/naming-conventions.html#trailing-question-mark-foo